Sentrii/src/Popup.jsx

```diff
@@ -248,24 +304,152 @@ export default function Popup() {
                     fontWeight: "bold",
                     fontSize: "large",
                     color: "#fff",
+                    paddingBottom: "10px",
                   }}
                 >
                   Warning! Could be malicious
                 </Text>
               )}
               {scanResult.stats.percentage >= 50 &&
                 scanResult.stats.percentage <= 75 && (
-                  <Text style={{ fontSize: "small", color: "#000" }}>
+                  <Text style={{ fontSize: "small", color: "#000" ,paddingBottom: "10px" }}>
                     Tread with caution
                   </Text>
                 )}
               {scanResult.stats.percentage > 75 && (
-                <Text style={{ color: "#fff" }}>All Good</Text>
+                <Text style={{ color: "#fff" , paddingBottom: "10px"}}>All Good</Text>
```

Comment:
All good maybe misleading because, score of 89 can still have high impact core issues like [this token](https://de.fi/scanner/contract/0xafd2bd087595a29204cbeb62048933927852775f?1)

Maybe we want to show "all good" only if there are no high impact issues.

Reply:
no worries, if score is > 75 but theres >0 high impact issues, it shall be labeled as "tread with caution" henceforth